app/src/organisms/ChooseRobotSlideout/index.tsx

```diff
@@ -316,7 +330,177 @@ export function ChooseRobotSlideout(
+                  return {
+                    ...parameter,
+                    value:
+                      runtimeParam.type === 'int'
+                        ? Math.round(e.target.valueAsNumber)
+                        : e.target.valueAsNumber,
+                  }
+                }
+                return parameter
+              })
+              if (setRunTimeParametersOverrides != null) {
+                setRunTimeParametersOverrides(clone)
+              }
+            }}
+          />
+        )
+      } else if (runtimeParam.type === 'boolean') {
+        return (
+          <Flex
+            flexDirection={DIRECTION_COLUMN}
+            key={runtimeParam.variableName}
+          >
+            <StyledText as="labelSemiBold" paddingBottom={SPACING.spacing8}>
+              {runtimeParam.displayName}
+            </StyledText>
+            <Flex
+              gridGap={SPACING.spacing8}
+              justifyContent={JUSTIFY_FLEX_START}
+              width="max-content"
+            >
+              <ToggleButton
+                toggledOn={runtimeParam.value as boolean}
+                onClick={() => {
+                  const clone = runTimeParametersOverrides.map(
+                    (parameter, i) => {
+                      if (i === index) {
+                        return {
+                          ...parameter,
+                          value: !parameter.value,
+                        }
+                      }
+                      return parameter
+                    }
+                  )
+                  if (setRunTimeParametersOverrides != null) {
+                    setRunTimeParametersOverrides(clone)
+                  }
+                }}
+                height="0.813rem"
+                label={runtimeParam.value ? t('on') : t('off')}
+                paddingTop={SPACING.spacing2} // manual alignment of SVG with value label
+              />
+              <StyledText as="p">
+                {runtimeParam.value ? t('on') : t('off')}
+              </StyledText>
+            </Flex>
+            <StyledText as="label" paddingTop={SPACING.spacing8}>
+              {runtimeParam.description}
+            </StyledText>
+          </Flex>
+        )
+      }
+    }) ?? null
+
+  const ENABLED_LINK_CSS = css`
+    ${TYPOGRAPHY.linkPSemiBold}
+    cursor: pointer;
+  `
+
+  const DISABLED_LINK_CSS = css`
+    ${TYPOGRAPHY.linkPSemiBold}
+    color: ${COLORS.grey50};
+    cursor: default;
+
+    &:hover {
+      color: ${COLORS.grey50};
+    }
+  `
+
+  const isRestoreDefaultsLinkEnabled =
+    runTimeParametersOverrides?.some(
+      parameter => parameter.value !== parameter.default
+    ) ?? null
+
+  const pageTwoBody =
+    runTimeParametersOverrides != null ? (
+      <Flex flexDirection={DIRECTION_COLUMN}>
+        <Flex justifyContent={JUSTIFY_END}>
+          <Link
+            textAlign={TYPOGRAPHY.textAlignRight}
+            css={
+              isRestoreDefaultsLinkEnabled
```

Comment:
if need to set null,

```suggestion
              isRestoreDefaultsLinkEnabled != null
```